articles/foreign-host-vitals-identity-provider-idp-user-information.md

```diff
@@ -0,0 +1,283 @@
+# Foreign host vitals: identity provider (IdP) user information
+
+![Import users from IdP to Fleet](../website/assets/images/articles/add-users-from-idp-cover-img.png)
+
+_Available in Fleet Premium._
+
+Fleet supports [Okta](#okta), [Microsoft Active Directory (AD) / Entra ID](#microsoft-entra-id), [Google Workspace](#google-workspace), [authentik](#google-workspace), as well as any other identity provider (IdP) that supports the [SCIM](https://scim.cloud/) standard.
+
+To add IdP host vitals, follow steps for your IdP.
+
+
+## Okta
+
+To connect Okta to Fleet, follow these steps:
+
+1. Head to Okta admin dashboard.
+2. Select **Applications > Applications** in the main menu, then select **Create App Integration**.
```

Comment:
```suggestion
2. In the main menu, select **Applications > Applications**, then select **Create App Integration**.
```

I think it's a good idea to start with the menu here. First go to the menu, then...